tests/UnifiedSpecTests/RunOnRequirement.php

```diff
@@ -31,38 +31,29 @@ class RunOnRequirement
 
     public const VERSION_PATTERN = '/^[0-9]+(\\.[0-9]+){1,2}$/';
 
-    /** @var string */
-    private $minServerVersion;
+    private ?string $minServerVersion = null;
 
-    /** @var string */
-    private $maxServerVersion;
+    private ?string $maxServerVersion = null;
 
-    /** @var array */
-    private $topologies;
+    private ?array $topologies = null;
 
-    /** @var stdClass */
-    private $serverParameters;
+    private ?object $serverParameters = null;
```

Comment:
Should we have kept `stdClass` here?

Reply:
It got tripped up by a `assertIsObject` check later in the file.